src/main/java/decodes/tsdb/TimeSeriesDb.java

```diff
@@ -1012,7 +1012,7 @@ public void releaseNewData(DataCollection dc, TimeSeriesDAI tsDAO)
 					+ "and ((" + curTime + " - DATE_TIME_LOADED) > " // curTimeName
 					+ inter ); //String.format(maxCompRetryTimeFrmt, maxRetries) + ")"); //
 			PreparedStatement updateFailedRetry = tcon.prepareStatement(
-				"update CP_COMP_TASKLIST set FAIL_TIME = ? where RECORD_NUM = ? and "
+				"update CP_COMP_TASKLIST set FAIL_TIME = " + curTime + " where RECORD_NUM = ? and "
```

Comment:
That needs to stay a parameter but just be included. 

A value like time, which will change every call, here will cause the the Oracle planner to constantly flush query causing huge performance problems beyond just OpenDCS usage.

Reply:
well, unless that particular value is a string and not time. that is one of the problems with the diff. they miss some context. either way I'll make sure the appropriate fix goes in.